### PR TITLE
Defined databse schema

### PR DESCRIPTION
**Pull request by NayanKaran**
Hi,

We have defined the schema for the clinic database according to the project requirements below.

Kindly verify.

### Project requirements
- Take a look at the following database diagram for a clinic:
clinic_diagram
- Create a file named `schema_based_on_diagram.sql` where you implement the database from the diagram.
- Join tables from many-to-many relationships might not appear in the diagram, but you still need them.
- Remember to add the FK indexes.

### CODE REVIEW

File: schema_based_on_diagram.sql [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+    total_price DECIMAL,
+    invoice_id INT,
+    treatment_id INT,
+    CONSTRAINT fk_invoices
+    FOREIGN KEY (invoice_id)
+    REFERENCES invoices(id)
+    CONSTRAINT fk_treatments
+    FOREIGN KEY (treatment_id)
+    REFERENCES treatments(id)
+);
```
Comment (sja-thedude) [inline schema_based_on_diagram.sql:52]: - [x] Kindly make sure that the foreign keys have indexes shown below:
```sql
CREATE INDEX ON medical_histories (patient_id);
CREATE INDEX ON invoices (medical_history_id);
CREATE INDEX ON invoice_items (invoice_id);
CREATE INDEX ON invoice_items (treatment_id);
CREATE INDEX ON medical_histories_has_treatments (medical_history_id);
CREATE INDEX ON medical_histories_has_treatments (treatment_id);
```

**Review by sja-thedude — CHANGES_REQUESTED** [review]
## Required Changes ♻️

Hi @NayanKaran & @ibrahim777764, This is @sja-thedude your code reviewer.

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

768070f3-bc9a-4aba-b8bc-25793e00ea1e_text

## Highlights
✔️ Professional Readme
✔️ Correct github
✔️ All linters passing

## Required 
- [ ] _Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me [@sja-thedude] in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: schema_based_on_diagram.sql (hunk 1 above)
Comment (NayanKaran) [reply schema_based_on_diagram.sql:52]: Hi @sja-thedude,

I have made the required changes. https://github.com/NayanKaran/clinic-database/pull/1/commits/6a4da931c9179010ae6c50563e9efd6f7640815b

Thank you for reviewing our code.

**Review by Kingstalux — APPROVED** [review]
Hi @team,
# Status: Approved :heavy_check_mark: :heavy_check_mark: :heavy_check_mark: 
68747470733a2f2f6d65646961322e67697068792e636f6d2f6d656469612f6c3366516333614356304871707a5763552f67697068792e676966
 
Good job adding indexes to your project :+1: Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**